go.mod

```diff
@@ -3,6 +3,12 @@ module github.com/warden-protocol/wardenprotocol
 go 1.22.5
 
 replace (
+	// evmos integration - need this replace when importing cosmos/rosetta pkg
+	cosmossdk.io/core => cosmossdk.io/core v0.11.0
+	// evmos integration
+	github.com/cosmos/cosmos-sdk => github.com/warden-protocol/cosmos-sdk v0.50.9-evmos-warden
+	// evmos + types for slinky
+	github.com/ethereum/go-ethereum => github.com/warden-protocol/go-ethereum v1.10.26-warden-evmos-rc2
```

Comment:
sad that we have to maintain these forks 😭

Reply:
we discussed with @MalteHerrmann that probably we can merge our changes in their fork